core/tests/protractor_utils/StoryEditorPage.js

```diff
@@ -264,88 +272,94 @@ var StoryEditorPage = function() {
   };
 
   this.expectTitleToBe = async function(title) {
+    await waitFor.visibilityOf(
+      storyTitleField,
+      'Story Title Field is taking too long to be visible');
     expect(await storyTitleField.getAttribute('value')).toEqual(title);
   };
 
   this.expectDescriptionToBe = async function(description) {
+    await waitFor.visibilityOf(
+      storyDescriptionField,
+      'Story Description Field taking too long to be visible');
     expect(await storyDescriptionField.getAttribute('value')).toEqual(
       description);
   };
 
   this.changeStoryTitle = async function(storyTitle) {
-    await storyTitleField.clear();
-    await storyTitleField.sendKeys(storyTitle);
+    await action.clear('Story title Field', storyTitleField);
+    await action.sendKeys(
+      'Story Title Field',
+      storyTitleField, storyTitle);
   };
 
   this.returnToTopic = async function() {
     await general.scrollToTop();
-    await returnToTopicButton.click();
+    await action.click(
+      'Return To Topic Button',
+      returnToTopicButton);
     await waitFor.pageToFullyLoad();
   };
 
   this.changeStoryDescription = async function(storyDescription) {
-    await storyDescriptionField.clear();
-    await storyDescriptionField.sendKeys(storyDescription);
+    await action.clear(
+      'Story Description Field', storyDescriptionField);
+    await action.sendKeys(
+      'Story Description Field',
+      storyDescriptionField, storyDescription);
   };
 
   this.changeStoryNotes = async function(richTextInstructions) {
-    await openStoryNotesEditorButton.click();
+    await action.click(
+      'Open Story Notes Editor Button', openStoryNotesEditorButton);
     var storyNotesEditor = await forms.RichTextEditor(
       notesEditor);
-    await storyNotesEditor.clear();
```

Comment:
This is actually fine because `storyNotesEditor` isn't an element. It's a rich text editor.